messenger.rst

```diff
@@ -619,7 +619,18 @@ times:
     process_name=%(program_name)s_%(process_num)02d
 
 Change the ``async`` argument to use the name of your transport (or transports)
-and ``user`` to the Unix user on your server. Next, tell Supervisor to read your
+and ``user`` to the Unix user on your server.
+
+If you use the Redis Transport, note that each worker needs a unique consumer name to
+avoid the same message being handled by multiple workers. One way to achieve this is
+to set an environment variable in the Supervisor configuration file, which you can
+then refer to in `messenger.yaml` (see Redis section above):
+
+.. code-block:: ini
+
+    environment=MESSENGER_CONSUMER_NAME=%(program_name)s_%(process_num)02d
```

Comment:
Why not use directly the "default" variable `MESSENGER_TRANSPORT_DSN`? 

```
environment=MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages/symfony/%(program_name)s_%(process_num)02d
```

It's less DRY, but the `MESSENGER_CONSUMER_NAME` is not mentioned anywhere else in the docs, it may be confusing.

Reply:
We've decided to merge this "as is" ... but we can change this later as you suggested. Thanks!